integration/nwo/weaver/util.go

```diff
@@ -179,7 +179,7 @@ func gzipDir(dir, dest string) error {
 	defer t.Close()
 	defer t.Flush()
 
-	err = filepath.Walk(dir, func(path string, info fs.FileInfo, err error) error {
+	err = filepath.Walk(dir, func(path string, info fs.FileInfo, _ error) error {
 		var f *os.File
```

Comment:
we should actually check if the err exists. See `Walk` docu (https://pkg.go.dev/path/filepath#Walk)

what about the following:
```go
err = filepath.Walk(dir, func(path string, info fs.FileInfo, err error) error {
    if err != nil {
        return fmt.Errorf("failed accessing path=%s: %v", path, err)
    }
    // if no error, we can open the file
    var f *os.File
    f, err = os.Open(path)
    ...
```

Reply:
I applied the change. If a sub-directory is not accessible for some reason (permission denied or corrupt data maybe), this would have been ignored before (and we would move on to the next sub-dirs), but not anymore. I guess this is never the case (and it's not a test case either), but just to be sure, is it OK to keep this change?

Reply:
> If a sub-directory is not accessible for some reason (permission denied or corrupt data maybe), this would have been ignored before (and we would move on to the next sub-dirs)

We still move on the next sub-dir in case of an error. Now we are just doing right away instead of trying to open the file even in the presence of an error.